Dockerfile.mayhem

```diff
@@ -0,0 +1,45 @@
+ARG NAMESPACE=
```

Comment:
I don't understand what all of the qemu-downloading commands are for. Could you elaborate?

Reply:
I'm honestly not sure; most of this is just copied from the project's `Dockerfile`.